src/logger/src/metrics.rs

```diff
@@ -769,7 +769,7 @@ struct SerializeToUtcTimestampMs;
 impl Serialize for SerializeToUtcTimestampMs {
     fn serialize<S: Serializer>(&self, serializer: S) -> Result<S::Ok, S::Error> {
         serializer.serialize_i64(
-            utils::time::get_time_ns(utils::time::ClockType::Monotonic) as i64 / 1_000_000,
+            utils::time::get_time_ns(utils::time::ClockType::Real) as i64 / 1_000_000,
```

Comment:
I'm not sure I agree here. There are PROs and CONs to each:

`ClockType::Monotonic` doesn't have a standardized base (on Linux though it's boot-time), so it might be hard to match timestamps in Firecracker logs to timestamps in other logs if other logs use other clocktype than monotonic.

`ClockType::Real` provides the machine's best guess as to current wall-clock, time-of-day time. The problem with this, is that it can jump forwards and backwards as the system time-of-day clock is changed, including by NTP and/or daylight savings. This means any time-jump event will make firecracker logs very confusing.

Reply:
I'm not sure which is better for our case. Can you check other components on the system to see what timestamps they use in their logs?

For example `systemd` seems to log both: https://unix.stackexchange.com/a/453258

Reply:
As discussed offline, one option would be to add both realtime and monotonic timestamps to the metric data. We'll further discuss in the team the options we have and add a conclusion soon.

Reply:
After looking here in more detail I see this is only related to a specific metric named `utc_timestamp_ms`. The value for this metric uses `monotonic` as time source so it's not really UTC time.

Given that metrics publishing happens every 60s using a monotonic timer, I think the PR is actually correct in simply changing value of this metric to use `realtime` source to provide correct UTC time.